pkg/setting/setting_unified_alerting.go

```diff
@@ -60,13 +63,37 @@ type UnifiedAlertingSettings struct {
 	EvaluationTimeout              time.Duration
 	ExecuteAlerts                  bool
 	DefaultConfiguration           string
+	Enabled                        bool
+	DisabledOrgs                   map[int64]struct{}
 }
 
 // ReadUnifiedAlertingSettings reads both the `unified_alerting` and `alerting` sections of the configuration while preferring configuration the `alerting` section.
 // It first reads the `unified_alerting` section, then looks for non-defaults on the `alerting` section and prefers those.
 func (cfg *Cfg) ReadUnifiedAlertingSettings(iniFile *ini.File) error {
 	uaCfg := UnifiedAlertingSettings{}
 	ua := iniFile.Section("unified_alerting")
+	uaCfg.Enabled = ua.Key("enabled").MustBool(false)
+
+	// if the old feature toggle ngalert is set, enable Grafana 8 alerts anyway
+	if !uaCfg.Enabled && cfg.FeatureToggles["ngalert"] {
```

Comment:
Can we print a warning here? Would be good to let the users know that they should remove the feature toggle and enable unified alerting explicitly. I would hope that support for this would go away two minor versions from now.